draft-terbu-sd-jwt-vc.md

```diff
@@ -31,8 +31,9 @@ organization="Authlete Inc. "
 .# Abstract
 
 This specification describes data formats as well as validation and processing
-rules to express Verifiable Credentials with JSON payload based on the SD-JWT
-format [@!I-D.ietf-oauth-selective-disclosure-jwt].
+rules to express Verifiable Credentials based on the Selective Disclosure
+for JWT and JWS with JSON payloads (SD-JWT) format [@!I-D.ietf-oauth-selective-disclosure-jwt].
```

Comment:
```suggestion
for JWT (SD-JWT) format [@!I-D.ietf-oauth-selective-disclosure-jwt].
```

Reply:
I agree that the "with JSON payloads" wording is clunky and should go.  All JWTs have JSON payloads, so we don't need to say so in the title.

Reply:
Also, see https://github.com/vcstuff/draft-terbu-sd-jwt-vc/pull/132, which requests a parallel change to the title.

Reply:
#132 should be addressed after we have clarity on direction whether #120 or #131. 
jfyi "Selective Disclosure for JWT and JWS with JSON payloads (SD-JWT)" is a new spec title for SD-JWT, and "JWS with JSON payloads" is meant to cater for JSON serialization which is not allowed in JWT>

Reply:
The full (and admittedly clunky) title of SD-JWT doesn't need to be spelled out here though.  (and ugh we should have capitalized Payloads in that title)  
```suggestion
for JWT (SD-JWT) [@!I-D.ietf-oauth-selective-disclosure-jwt] format.
```